### DISCUSSION TITLE
Highlighted Items Import File

### DISCUSSION THREAD

**Discussion by bearryj** [runelite/runelite#17172] [Ideas]
I would like to be able to import a file that list every item in the game and assign them a highlight value tied with the lootbeam tiers. 

Or just a list of high tier items like raid item drops and rare drop table items.

This would make lootbeams usable throughout the entire game.

**Comment by geheur** [comment]
why can't you just use regular gp-valued lootbeams?

**Reply by bearryj** [reply]
For one, you can only have one tier highlighted for lootbeams. And the only way to have other items highlighted is to input them into the highlighted list. It's clunky, just put it in a single import file where each item can either be tied to a gp value or drop table value. This way you could significantly change the visual aesthetic of drops in the game. Rather than currently, where we have maybe 10-20 items manually written into the highlight section. Which brings up another problem, for instance, my hidden items section is pretty long and it extends the settings section quite a bit because it doesn't hide after a certain length. Just another reason to implement the ability to import your item list.
